### PR TITLE
Move event type descriptions from Terminology to Workflow Syntax

### PR DESCRIPTION
**Pull request by not-my-profile**
I expect a syntax reference to not just list the allowed values but to define them as well.

### CODE REVIEW

**Review by xoxys — APPROVED** [review]

File: docs/docs/20-usage/15-terminology/index.md [hunk 1]
```diff
@@ -33,16 +33,6 @@
 
 ![Relation between pipelines, workflows and steps](pipeline-workflow-step.svg)
 
-## Pipeline events
```
Comment (anbraten) [inline docs/docs/20-usage/15-terminology/index.md:36]: Would be nice to have a link to the new location here

Comment (not-my-profile) [reply docs/docs/20-usage/15-terminology/index.md:36]: I'd say strictly speaking the syntax values aren't terminology, e.g. `pull_request_closed` isn't really a term.

Comment (6543) [reply docs/docs/20-usage/15-terminology/index.md:36]: we should add an link to L17 @not-my-profile

File: docs/docs/20-usage/15-terminology/index.md [hunk 2]
```diff
@@ -7,18 +7,18 @@
 - **Server**: The component of Woodpecker that handles webhooks from forges, orchestrates agents, and sends status back. It also serves the API and web UI for administration and configuration.
 - **Agent**: A component of Woodpecker that executes [pipelines][Pipeline] (specifically one or more [workflows][Workflow]) with a specific backend (e.g. [Docker][], Kubernetes, [local][Local]). It connects to the server via GRPC.
 - **CLI**: The Woodpecker command-line interface (CLI) is a terminal tool used to administer the server, to execute pipelines locally for debugging / testing purposes, and to perform tasks like linting pipelines.
-- **Pipeline**: A sequence of [workflows][Workflow] that are executed on the code. [Pipelines][Pipeline] are triggered by events.
-- **Workflow**: A sequence of steps and services that are executed as part of a [pipeline][Pipeline]. Workflows are represented by YAML files. Each [workflow][Workflow] has its own isolated [workspace][Workspace], and often additional resources like a shared network (docker).
+- **[Pipeline][Pipeline]**: A sequence of [workflows][Workflow] that are executed on the code. Pipelines are triggered by events.
```
[the remark this answers is not in the document]
Comment (not-my-profile) [reply docs/docs/20-usage/15-terminology/index.md:10]: I didn't change any capitalization and all terms are currently consistently capitalized. I consider changing the current capitalization to be out of the scope of this MR.

**Comment by 6543** [comment]
merged :mage:

**Pull request merged**